src/rooms/RestartRoom.ts

```diff
@@ -11,11 +11,36 @@ export class RestartRoom extends Room<RestartRoomState> {
   maxClients = 8;
   LOBBY_CHANNEL = "$restartlobby";
 
+  //Constructor for disposeTimer
+  private disposeTimer: NodeJS.Timeout | null = null;
+
+  constructor() {
+    super();
+    this.startDisposeTimer = this.startDisposeTimer.bind(this);
+  }
+
+  private startDisposeTimer() {
+    this.disposeTimer = setTimeout(() => {
+      if (this.clients.length <= 1) {
+        this.onDispose(); // Disconnect clients and dispose if conditions are met
```

Comment:
Don't run onDispose() as it is just a callback and might not deallocate memory. Ref: https://docs.colyseus.io/server/room/#ondispose

Instead, use Room#disconnect. Ref: https://docs.colyseus.io/server/room/#disconnect